website/docs/d/msk_cluster.html.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: "aws"
+page_title: "AWS: aws_msk_cluster"
+sidebar_current: "docs-aws-datasource-msk-cluster"
```

Comment:
This documentation page is missing a sidebar link in the data sources section of `website/aws.erb`